packages/uni-virtualizer/src/lib/polyfillLoaders/EventTarget.js

```diff
@@ -0,0 +1,16 @@
+let ET;
+
+export default async function EventTarget() {
+    return ET || init();
+}
+
+async function init() {
+    ET = window.EventTarget;
+    try {
+        new ET();
+    }
+    catch {
+        ET = (await import('event-target-shim')).EventTarget;
```

Comment:
We could remove this dependency by implementing the `VirtualScroller` | `Layout` communication layer with callbacks instead.